open_event/views/views.py

```diff
@@ -196,6 +196,18 @@ def get_event_version(event_id):
     return jsonify({"version": []})
 
 
+@app.route('/api/v1/event/<event_id>/sessions/<string:session_name>', methods=['GET'])
+@cross_origin()
+def get_sessions_at_event(event_id, session_name):
+    sessions=Session.query.filter(Session.event_id == event_id,Session.name == session_name)
```

Comment:
Let's us .contains() on the session name, so people can search for all sessions containing a certain word/etc.